app/assets/javascripts/application.js

```diff
@@ -400,3 +400,27 @@ function setPowerState(item, status){
   power_actions.hide();
   $('[rel="twipsy"]').tooltip();
 }
+
+function disableButtonToggle(item) {
+  var isActive = item.classList.contains("active");
+  var formControl = $(item.parentElement.parentElement).children('.form-control');
```

Comment:
`children` returns jQuery object so there's no need to use `$(formControl)` later.